File: Square_of_a_Sorted_Array/SSArray.java
class Solution {
    public int[] sortedSquares(int[] nums) {
        int i; 
        int j;
        int index = nums.length - 1;
        int start = 0;
        int end = nums.length - 1;
        int[] myArr = new int[index + 1];
        while(start <= end)
        {
            i = nums[start] * nums[start];
            j = nums[end] * nums[end];
            if(i > j)
            {
                myArr[index--] = i;
                start++;
            }
            else
            {
                myArr[index--] = j;
                end--;
            }
        }
        
        return myArr;
    }
}

File: Square_of_a_Sorted_Array/SSArray.cs
public class Solution {
    public int[] SortedSquares(int[] nums) {
        int i, x;
        for(i = 0; i < nums.Length; i++)
        {
            x = nums[i] * nums[i];
            nums[i] = x;
        }
        Array.Sort(nums);
        return nums;
    }
}